padl/transforms.py

```diff
@@ -328,7 +381,7 @@ def pd_post_load(self, path: Path, i: int):
     def pd_zip_save(self, path: Union[Path, str], force_overwrite: bool = False):
         """Save the transform to a zip-file at *path*.
 
-        The file's name should end with '.padl'. If no extension is given, it will be added
+        The file's pd_name should end with '.padl'. If no extension is given, it will be added
```

Comment:
Change `pd_name` back to `name`

Reply:
It looks like this happened to the docstrings throughout the file. Please change back.